src/main/io/serial_cli.c

```diff
@@ -2617,16 +2643,16 @@ static void cliBeeper(char *cmdline)
 }
 #endif
 
-static void printMap(uint8_t dumpMask, const rxConfig_t *defaultRxConfig)
+static void printMap(uint8_t dumpMask, const rxConfig_t *rxConfig, const rxConfig_t *defaultRxConfig)
 {
```

Comment:
Probably best to add a NULL check here as well for `defaultRxConfig`, lest it be used in the same pattern as the other `print...` functions at some point in the future.